src/KafkaFlow/Middlewares/TypedHandler/Configuration/TypedHandlerConfigurationBuilder.cs

```diff
@@ -1,4 +1,4 @@
-namespace KafkaFlow.TypedHandler
+namespace KafkaFlow.Middlewares.TypedHandler.Configuration
```

Comment:
Given that this file only has configuration methods utilized during setup, it's appropriate to adopt the root namespace `KafkaFlow`.